charts/consul/templates/NOTES.txt

```diff
@@ -5,8 +5,8 @@ Your release is named {{ .Release.Name }}.
 
 To learn more about the release, run:
 
-  $ helm status {{ .Release.Name }}
-  $ helm get all {{ .Release.Name }}
+  $ helm status {{ .Release.Name }} {{- if .Release.Namespace }} -n {{ .Release.Namespace }}{{ end }}
+  $ helm get all {{ .Release.Name }} {{- if .Release.Namespace }} -n {{ .Release.Namespace }}{{ end }}
```

Comment:
```suggestion
  $ helm status {{ .Release.Name }} {{- if .Release.Namespace }} --namespace {{ .Release.Namespace }}{{ end }}
  $ helm get all {{ .Release.Name }} {{- if .Release.Namespace }} --namespace {{ .Release.Namespace }}{{ end }}
```
It would probably be helpful to use the long form of this flag so that its very clear that this lookup is happening within a specific Kubernetes namespace.